lib/containers/utils.pm

```diff
@@ -179,4 +179,13 @@ sub test_built_img {
     assert_script_run("rm -rf /root/templates");
 }
 
+sub is_sle_container_supported {
+    my $is_osd = get_var('OPENQA_HOSTNAME', '') =~ /suse.de/;
+    my $is_o3  = get_var('OPENQA_HOSTNAME', '') =~ /opensuse.org/;
+    return 0 if $is_o3;
+    # Sometimes host is not SLE in osd
+    return 0 if $is_osd && check_os_release('opensuse', 'PRETTY_NAME');
+    return 1;
+}
```

Comment:
false if you try to test sle base container on o3 which run opensuse distros 
false if you try to test sle base container on osd but host is not sle
true otherwise

i did not know about .cz/.asia

Reply:
I like the previous version. This is certainly not ideal.

Reply:
i updated it to regex .cz as well.

Reply:
@mloviska where do you see the problem with this approach. for me it is actually better and to be honest i got confused with `is_sle` in some point. This makes it more clear, what the conditions

Reply:
As long as sle container is supported only on sle host I do not see any need to overengineer it. Making use of `is_sle` in sense whether the container is sle based ( because that is our main test object ) is perfectly fine. This way you are blocking all runs outside those 2 instances. Again, less is more :)

Reply:
And mainly it has nothing to do where I run the test

Reply:
OSD verification works. you can see that https://openqa.suse.de/tests/4963783#step/docker_image/5 does not block anything if host supports the sle base. i am still waiting VR from o3. 
you need a clear way to distinguish this requirement(where is supported and where not). `is_sle` is not clear at all as it is not "designed" for such situation.

Reply:
> false if you try to test sle base container on o3 which run opensuse distros
> false if you try to test sle base container on osd but host is not sle
true otherwise

Can you please those two lines as comments?